requirements.txt

```diff
@@ -2,3 +2,4 @@ nose==1.3.7
 numpy==1.9.2
 scipy==0.16.0
 matplotlib==1.4.3
+sklearn==0.17.0
```

Comment:
Version 0.17.0 is released on November, so older vision might work. Try Version 0.16.0